src/test/java/com/google/api/codegen/testdata/fakebilling_gapic.yaml

```diff
@@ -20,11 +23,122 @@ license_header:
   license_file: license-header-apache-2.0.txt
 interfaces:
 - name: google.fakebilling.v1.BillingAccounts
+  lang_doc:
+    java: Configure billing accounts.
+  collections:
+  - name_pattern: billingAccounts/{billingAccountsId}
+    entity_name: billingAccountsId
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  - name: pull
```

Comment:
`pull` & `non_idempotent` are not used, so I think they should be removed (same thing further down too)